en/README.md

```diff
@@ -21,7 +21,7 @@ We hope that we'll be able to make you love technology as much as we do!
 
 ## What will you learn during the tutorial?
 
-Once you've finished the tutorial, you will have a simple, working web application: your own blog. We will show you how to put it online, so others will see your work!
+Once you've finished the tutorial, you will have a working web application: your own blog. We will show you how to put it online, so others will see your work!
```

Comment:
I think in this case simple makes sense. We don't want simple when we want to say doing something is simple. While indicating that the website is simple is true.

Reply:
basic, minimal or small might be better? less implications about complexity.

Reply:
Makes sense! Thank you :)